infrastructure/compute-to-data-minikube.md

```diff
@@ -129,115 +164,89 @@ volumeBindingMode: Immediate
 
 For more information, please visit https://kubernetes.io/docs/concepts/storage/storage-classes/
 
-#### Download and Configure Operator Service
+If you need to use your own classes, you will need to edit 'operator_engine/kubernetes/operator.yml'.
 
-Open a new terminal and run the command below.
+#### Setup C2D Orchestrator
 
-```bash
-git clone https://github.com/oceanprotocol/operator-service.git
-```
+C2D Orchestrator (aka operator-service) has two main functions:
+ - First, it's the outside interface of your C2D Cluster to the world. External components(like Provider) are calling APIs exposed by this
+ - Secondly, operator-service manages multiple environments and sends the jobs to the right environment.
 
-Edit `operator-service/kubernetes/postgres-configmap.yaml`. Change `POSTGRES_PASSWORD` to a nice long random password.
+Edit `operator-service/kubernetes/deployment.yaml`. Change `ALLOWED_ADMINS` to a nice long random password.
 
-Edit `operator-service/kubernetes/deployment.yaml`. Optionally change:
+Let's deploy C2D Orchestrator.
 
-* `ALGO_POD_TIMEOUT`
-* add `requests_cpu`
-* add `requests_memory`
-* add `limits_cpu`
-* add `limits_memory`
+```bash
+kubectl config set-context --current --namespace ocean-operator
+kubectl apply -f operator-service/kubernetes/deployment.yaml
+```
 
-```yaml
+Now, let's expose the service.
 
----
-spec:
-  containers:
-    - env:
-        - name: requests_cpu
-          value: "4"
-        - name: requests_memory
-          value: "8Gi"
-        - name: limits_cpu
-          value: "8"
-        - name: limits_memory
-          value: "15Gi"
-        - name: ALGO_POD_TIMEOUT
-          value: "3600"
+```bash
+kubectl expose deployment operator-api --namespace=ocean-operator --port=8050
 ```
 
-#### Download and Configure Operator Engine
+You can run a port forward in a new terminal (see below) or create your ingress service and setup DNS and certificates (not covered here):
 
 ```bash
-git clone https://github.com/oceanprotocol/operator-engine.git
+kubectl -n ocean-operator port-forward svc/operator-api 8050
 ```
 
-Check the [README](https://github.com/oceanprotocol/operator-engine#customize-your-operator-engine-deployment) section of the operator engine to customize your deployment.
+Alternatively you could use another method to communicate between the C2D Environment and the provider, such as an SSH tunnel.
```

Comment:
It could also be useful here to have a link to an external resource that explains more on how to use an ssh tunnel.